charts/vela-core/templates/defwithtemplate/task.yaml

```
@@ -36,6 +37,9 @@ spec:
       	// +short=i
       	image: string
       
+      	// +usage=Define job restart policy, the value should be Never or OnFailure. By default, it's Never.
+      	restartPolicy: *"Never" | string
```

```suggestion
      	restart: *"Never" | string
```

Let's name it `restart` for short

---

Ok, long parameters are a bit cumbersome